net/dnscrypt-proxy/Makefile

```diff
@@ -51,7 +51,7 @@ endef
 define Package/dnscrypt-proxy-resolvers
```

Comment:
`dnscrypt-proxy-resolvers` package has it's own version with date and git commit hash. Updating other packages version not necessary.

P.S. I'll update this commit, since it was 28 days ago.